kitsune/flagit/jinja2/flagit/content_moderation.html

```diff
@@ -57,6 +66,29 @@ <h3 class="sumo-card-heading"><br>{{ _('Update Status:') }}</h3>
       options=products,
       selected_filter=selected_product
   ) }}
+  {{ filter_dropdown(
+      form_id='assignee-filter-form',
+      select_id='flagit-assignee-filter',
+      label='Filter by assignee:',
+      name='assignee',
+      default_option='All assignees',
+      options=assignees,
+      selected_filter=selected_assignee
+  ) }}
+  <div id="my-queue-tools" data-current-username="{{ current_username }}"
+       {% if not (selected_assignee and (selected_assignee == current_username)) %} hidden{% endif %}>
+    <label>{{ _('Manage my queue') }}:</label>
+    <form id="my-queue-assign" hx-post="" hx-target="#flagged-queue" hx-select="#flagged-queue">
```

Comment:
I believe this ID `id` is not used anywhere.

Reply:
Good catch!